src/components/KModalFullscreen/KModalFullscreen.vue

```diff
@@ -260,40 +261,40 @@ $fullscreen-modal-padding: 64px;
 }
 
 .k-modal-fullscreen-header {
-  background-color: var(--white);
-  border-bottom: 1px solid var(--grey-300);
+  background-color: var(--white, var(--kui-color-background, $kui-color-background));
+  border-bottom: var(--kui-border-width-10, $kui-border-width-10) solid var(--grey-300, var(--kui-color-border-neutral-weak, $kui-color-border-neutral-weak));
   display: flex;
   flex-direction: column;
-  padding: var(--spacing-lg) 0;
+  padding: var(--spacing-lg, var(--kui-space-80, $kui-space-80)) var(--kui-space-0, $kui-space-0);
   position: fixed;
-  top: 0;
+  top: var(--kui-space-0, $kui-space-0);
```

Comment:
issue: we don't use space tokens for positioning

This can be reverted. Same comment for all instances of positioning